### DISCUSSION TITLE
38C3 assembly, meetings, talks or workshops?

### DISCUSSION THREAD

**Discussion by dllud** [markqvist/Reticulum#593] [General]
Hi everyone,

The [38th Chaos Communication Congress (38C3)](https://events.ccc.de/congress/2024/infos/startpage.html) is approaching, with preparations already going on.

On the previous edition @Erethon organized a community meeting (check out #386). Unfortunately he may not attend this year.
I am reaching out to know if other knowledgeable Reticulum community members would be available to host either meetings, introductory talks, workshops or even an assembly. 
(An assembly is just a table/space where community members hang around to hack together, show demos, answer questions from the general public, and likes.)

I am one of the organizers of the [Critical Decentralisation Cluster (CDC)](https://decentral.community/). For those not familiar with The Congress, a cluster (now also dubbed habitat) is a physical area and grouping of different assemblies, under a common theme.
At 38C3, the CDC will grow, besides more workshop areas and tables for assemblies, we will also take part on running one of the community stages (for talks).

Reticulum has been under my (and others) radar for a while now and it aligns perfectly with the CDC's mission. As we now have the necessary space, it would be awesome if you could join and teach everyone more about Reticulum.
Usually 2 people suffice to run an assembly with ease.

My knowledge of Reticulum is super limited and only theoretical. But I (and other CDC members) will be available to help on the organizational aspects (assembly/talk/workshop registration, build-up, running it, etc.).

Cheers!

**Comment by resiliencetheatre** [comment]
I might be interested to join the fun with my Edgemap project! My experience and previous talks were given mostly on restricted events, but now my project is open source and therefore might be interest of group. I've been in 'decentralized' world for decades and created resilience strategies and implementations to various governmental entities. So not only bits - but some stories along the technology. More information about Edgemap at my [Github profile](https://github.com/resiliencetheatre).

**Reply by dllud** [reply]
Nice! I think you should present Edgemap as a talk. The official C3's Call for Participation (CfP) for the Community Stages will open in a couple of days. I will came back with a link once it is live.
In case you also wish to host a workshop, you may register it for CDC's workshop areas at: https://pretalx.riat.at/38c3/cfp

**Comment by dllud** [comment]
The CfP for the Community Stages is now live:
https://content.events.ccc.de/cfp/38c3-community-stages/
https://cfp.cccv.de/38c3-community-stages/cfp

@resiliencetheatre I think you should submit a talk about Edgemap there.

Too all others: talks both presenting an overview of Reticulum, or getting into the weeds of it, will certainly be gladly welcomed. If you're attending C3 and have the knowledge please come and share it.

A few people from the [Critical Decentralisation Cluster](https://decentral.community/) will bring LoRa hardware to play with. If one or two people with a good grasp of Reticulum join in we can help you structure an assembly and workshops.

**Reply by gretel** [reply]
https://matrix.to/#/#cdc-public:dod.ngo

**Comment by markqvist** [comment]
Now that it's confirmed, I'm happy to say that I'll be attending as well. There will be a Reticulum assembly, part of the Critical Decentralisation Cluster, so for anyone there, please come by and say hello! Will be giving a talk about Reticulum as well.
